homeassistant/components/volkswagencarnet/__init__.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library, 3rd party and homeassistant import groups. Also sort imports according to PEP8:
https://www.python.org/dev/peps/pep-0008/#imports